### PR TITLE
Itinerary Body Caret Improvements

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
The itinerary body previously showed carets when they were inoperable. This PR ensures they are only shown when it makes sense

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3241,23 +3241,15 @@
   resolved "https://registry.yarnpkg.com/@open-draft/until/-/until-1.0.3.tgz#db9cc719191a62e7d9200f6e7bab21c5b848adca"
   integrity sha512-Aq58f5HiWdyDlFffbbSjAlv596h/cOnt2DO1w3DOC7OJ5EHs0hd/nycJfiu9RJbT6Yk6F1knnRRXNSpxoIVZ9Q==
 
-"@opentripplanner/core-utils@11.1.2":
-  version "11.1.2"
-  resolved "https://registry.yarnpkg.com/@opentripplanner/core-utils/-/core-utils-11.1.2.tgz#a99d1fa4fb1f587d58fae8fcfd70a3bfe4eed424"
-  integrity sha512-Rn1tBm5F+nt/A4/0cpq6cyNTprSsxfFslujMoZ4P4r6fZ7YTx0d25di+MZ/CZgSlCzHJIiGAfi8DsfJ7yStcTA==
+"@opentripplanner/geocoder@^1.4.2":
```
Comment (amy-corson-ibigroup) [inline yarn.lock:3244]: what's up here!

Comment (miles-grant-ibigroup) [reply yarn.lock:3244]: I don't know!

File: packages/itinerary-body/src/TransitLegBody/index.tsx [hunk 2]
```diff
@@ -300,7 +300,9 @@ class TransitLegBody extends Component<Props, State> {
                     alertCount: alerts.length
                   }}
                 />
-                <S.CaretToggle expanded={alertsExpanded} />
+                {!shouldOnlyShowAlertsExpanded && (
```
Comment (amy-corson-ibigroup) [inline packages/itinerary-body/src/TransitLegBody/index.tsx:303]: Wait doesn't `shouldOnlyShowAlertsExpanded` just impact how the alerts are rendered? The caret should still be there if users want to collapse them after render, right?

**Review by amy-corson-ibigroup — COMMENTED** [review]
Quick q about the alerts!

File: packages/itinerary-body/src/TransitLegBody/index.tsx (hunk 2 above)
Comment (miles-grant-ibigroup) [reply packages/itinerary-body/src/TransitLegBody/index.tsx:303]: That behavior doesn't work right now

**Review by amy-corson-ibigroup — APPROVED** [review]

File: yarn.lock [hunk 3]
```diff
@@ -3241,23 +3241,15 @@
...
-    "@opentripplanner/geocoder" "^1.4.2"
-    "@styled-icons/foundation" "^10.34.0"
-    "@turf/along" "^6.0.1"
-    chroma-js "^2.4.2"
-    date-fns "^2.28.0"
-    date-fns-tz "^1.2.2"
-    graphql "^16.6.0"
-    lodash.clonedeep "^4.5.0"
-    lodash.isequal "^4.5.0"
-    qs "^6.9.1"
```
Comment (daniel-heppner-ibigroup) [inline yarn.lock:3260]: wow good job cleaning up the dependencies with this pr

**Review by daniel-heppner-ibigroup — APPROVED** [review]

**Pull request merged**